.travis.yml

```diff
@@ -43,3 +43,14 @@ install:
 
 script:
     - . ./utils/travis/steps/script.sh
+
+notifications:
```

Comment:
It would be best if you rewrote history a bit to remove this mapping (yes, that's the right YAML term) being removed and subsequently restored.

Reply:
Alright.  Not sure how to do that at the moment, but I'll figure it out tomorrow.

Reply:
This should be fine now.